### PR TITLE
chore(data-list): remove card from demos

### PR DESCRIPTION
**Pull request by andrew-ronaldson**
Closes https://github.com/patternfly/patternfly/issues/6795

### CODE REVIEW

**Review by mcoker — COMMENTED** [review]
LGTM! Just a quick glance - I noticed these table demos are in a card, too - should we update them?

https://patternfly-pr-6811.surge.sh/components/table/html-demos/sortable/
https://patternfly-pr-6811.surge.sh/components/table/html-demos/expandable/
https://patternfly-pr-6811.surge.sh/components/table/html-demos/compact/
https://patternfly-pr-6811.surge.sh/components/table/html-demos/compound-expansion/
https://patternfly-pr-6811.surge.sh/components/table/html-demos/loading-state-demo/
https://patternfly-pr-6811.surge.sh/components/table/html-demos/empty-state/
https://patternfly-pr-6811.surge.sh/components/table/html-demos/static-bottom-pagination/
https://patternfly-pr-6811.surge.sh/components/table/html-demos/column-management-modal/
https://patternfly-pr-6811.surge.sh/components/table/html-demos/sticky-header/

**Review by mcoker — APPROVED** [review]
L👏 G👏 T👏 M👏

File: src/patternfly/demos/Table/examples/Table.md [hunk 1]
```diff
@@ -33,21 +33,19 @@ import './Table.css'
...
+        toolbar-template--HasToggleGroup=true
+        toolbar-template--HasSearchFilter=true
+        toolbar-template--HasOverflowMenu=true
+        toolbar-template--HasOverflowMenuSecondButton=true
+        toolbar-template--HasIconButtonGroup=true
+        toolbar-template--HasSortButtonMobile=true
+      }}
+    {{> table-sortable-table}}
+    {{> table-pagination-footer}}
+    {{/page-main-section}}
```
Comment (mattnolting) [inline src/patternfly/demos/Table/examples/Table.md:48]: NIT: 
```suggestion
  {{/page-main-section}}
```

**Review by mattnolting — APPROVED** [review]
LPTM! Thanks Andrew!

**Pull request merged**